SQRLDotNetClientUI/ViewModels/NewIdentityViewModel.cs

```diff
@@ -74,12 +74,8 @@ public async void GenerateNewIdentity()
             }
             else
             {
-                var messageBoxStandardWindow = MessageBoxManager.GetMessageBoxStandardWindow(
-                    _loc.GetLocalizationValue("ErrorTitleGeneric"),
-                    _loc.GetLocalizationValue("PasswordsDontMatchErrorMessage"),
-                    ButtonEnum.Ok, Icon.Error);
-
-                await messageBoxStandardWindow.ShowDialog(_mainWindow);
+                
+                await new Views.MessageBox(_loc.GetLocalizationValue("ErrorTitleGeneric"), _loc.GetLocalizationValue("PasswordsDontMatchErrorMessage"), MessageBoxSize.Medium, MessageBoxButtons.OK, MessageBoxIcons.ERROR).ShowDialog<MessagBoxDialogResult>(_mainWindow);
```

Comment:
Just a matter of taste probably, but I very much prefer spreading such long function calls with a lot of parameters out over several lines. This vastly improves readability and maintainability. For this review alone, if I were to look at every single message box call, I'd have to scroll horizontally way too often just to be able to look at all the code.

So I would prefer this:

```C#
await new Views.MessageBox(
    _loc.GetLocalizationValue("ErrorTitleGeneric"), 
    _loc.GetLocalizationValue("PasswordsDontMatchErrorMessage"), 
    MessageBoxSize.Medium, MessageBoxButtons.OK, MessageBoxIcons.ERROR)
    .ShowDialog<MessagBoxDialogResult>(_mainWindow);
```
over this:

```C#
await new Views.MessageBox(_loc.GetLocalizationValue("ErrorTitleGeneric"), _loc.GetLocalizationValue("PasswordsDontMatchErrorMessage"), MessageBoxSize.Medium, MessageBoxButtons.OK, MessageBoxIcons.ERROR).ShowDialog<MessagBoxDialogResult>(_mainWindow);
```

Just sharing my thoughts here, please feel free to ignore any of this.

Reply:
You are right, Fixed all instances